src/views/Terms.vue

```diff
@@ -13,7 +13,7 @@
           </p>
 
           <p>
-            The service on http://youdonotneedacapetobeahero.com/ is provided by Superhero.com LVC, e-mail for correspondence: superherowallet@protonmail.com.
+            The service on http://superhero.com/ is provided by Superhero.com LVC, e-mail for correspondence: superherowallet@protonmail.com.
```

Comment:
```suggestion
            The service on https://beta.superhero.com/ is provided by Superhero.com LVC, e-mail for correspondence: superherowallet@protonmail.com.
```

Beta still.

Reply:
IMHO we could use the domain name only, as it is all under control of a single legal entity